packages/overture-schema-places-theme/pyproject.toml

```diff
@@ -27,4 +27,4 @@ path = "src/overture/schema/places/__about__.py"
 packages = ["src/overture"]
 
 [project.entry-points."overture.models"]
-"places.place" = "overture.schema.places.place.models:Place"
+"places.place" = "overture.schema.places:Place"
```

Comment:
Note that I made a similar change in the CLI PR. In that case, it may refer to an indirect alias, but either way, the intent is to register the shortest useful import for the class (because that path will be displayed when listing available types).